cpp/include/cudf/types.h

```diff
@@ -168,8 +181,16 @@ typedef struct gdf_context_{
   int flag_distinct;            /**< for COUNT: DISTINCT = 1, else = 0 */
   int flag_sort_result;         /**< When method is GDF_HASH, 0 = result is not sorted, 1 = result is sorted */
   int flag_sort_inplace;        /**< 0 = No sort in place allowed, 1 = else */
+  int flag_groupby_include_nulls; 
```

Comment:
You can use a `bool` here. The other ones use `int` because of legacy CFFI, but you don't have to worry about that anymore.